Gopkg.toml

```diff
@@ -0,0 +1,74 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+ignored = ["github.com/Sirupsen/logrus", "github.com/gravitational/license/generate*", "github.com/gravitational/license/authority*", "github.com/cloudflare/cfssl*"]
+
+[prune]
+  go-tests = true
+  unused-packages = true
+  non-go = true
+
+[[override]]
+  name = "github.com/alecthomas/template"
+  branch = "master"
+
+[[override]]
+  name = "github.com/alecthomas/units"
+  branch = "master"
+
+[[override]]
+  name = "github.com/cenkalti/backoff"
+  version = ">=1.0.0, <=19.0.0-g5d150e7"
+
+[[constraint]]
+  name = "github.com/gravitational/kingpin"
+  revision = "742f2714c145075a68df26ccc25bc05545310f3a"
+
+[[override]]
+  name = "github.com/prometheus/procfs"
```

Comment:
There're many Teleport-specific stuff here. I think it's not necessary to put all of this here. Lets just leave here only direct dependencies of plugins

Reply:
I left only teleport-specific `override`s here that affect nested dependencies. If you delete those overrides and try to do something like `dep ensure -add some_package`, `dep` will break versions compatibility.

`Gopkg.toml` is not like `Gemfile`, it contains mostly rules, not requirements. If it contains `override` or `constraint` of some package, this doesn't mean these package **will** be vendored.